src/main/webapp/index.html

```diff
@@ -21,24 +25,52 @@
       </li>
       <li class="nav-item">
         <a class="nav-link" href="intro.html">About</a>
-      </li>  
-    </ul>  
-    <div id="youtube-data">  
-      <div class="form-group">  
-        <form onsubmit="return false">
-          <p>Enter a YouTube video link:</p>
-          <input id="link-input" name="link-input" required pattern="https?://www.youtube.com/watch\?v=.+"/>
-          <small id="url-help" class="form-text text-muted">Format: https://youtube.com/watch?v=...</small>
-          <p>Enter the maximum number of comments to be analyzed: </p>
-          <input type="range" id="num-comments-input" name="num-comments-input" min="100" max="1000" value = "100" step="10"/>
-          <small id="slider-output" name = "slider-output">100</small>
-          </br>
-          <input type="submit" id="submit-btn"/>
-          </br>
-        </form>
+      </li>
+    </ul>
+    <div id="youtube-data">
+      <div class="form-group">
+        <nav>
+          <div class="nav nav-tabs" id="nav-tab" role="tablist">
+            <a class="nav-item nav-link active" id="nav-keyword-tab" data-toggle="tab" href="#nav-keyword" role="tab" aria-controls="nav-keyword" aria-selected="true">Keyword Search</a>
+            <a class="nav-item nav-link" id="nav-videolink-tab" data-toggle="tab" href="#nav-videolink" role="tab" aria-controls="nav-videolink" aria-selected="false">Video Link</a>
+          </div>
+        </nav>
+        <div class="tab-content" id="nav-tabContent">
+          <div class="tab-pane fade show active" id="nav-keyword" role="tabpanel" aria-labelledby="nav-keyword-tab">
+            <br>
+            <p>Enter a YouTube video keyword:</p>
+            <input type="text" id="search-input">
+            <input name="Search" type="submit" value="Search!" id="submit-button" onclick="getVideoResults()"/>
```

Comment:
This one should not have the submitBtn class because submitBtn is related in script.js onButtonPress function. This one only requires getVideoResults() to be called

Reply:
Ok sounds good.